Python/Product/TestAdapter.Executor/Pytest/TestDiscovererPytest.cs

```diff
@@ -41,8 +41,7 @@ internal class TestDiscovererPytest : IPythonTestDiscoverer {
         public void DiscoverTests(
             IEnumerable<string> sources,
             IMessageLogger logger,
-            ITestCaseDiscoverySink discoverySink,
-            Dictionary<string, PythonProjectSettings> sourceToProjectSettings
```

Comment:
Previous change i started to pass this Dictionary<string, PythonProjectSettings> sourceToProjectSettings around everywhere.  now i realize i dont need to do this.

I can add the TestContainer Source to the PythonProjectSettings class